src/services/OswServiceTaskMemMonitor.cpp

```diff
@@ -0,0 +1,52 @@
+#include "./services/OswServiceTaskMemMonitor.h"
+
+#include "osw_hal.h"
+#include "services/OswServiceManager.h"
+
+void OswServiceTaskMemMonitor::setup(OswHal* hal) { OswServiceTask::setup(hal); }
+
+void OswServiceTaskMemMonitor::loop(OswHal* hal) {
+  unsigned core0 = uxTaskGetStackHighWaterMark(nullptr);
```

Comment:
I think I could benefit from some comments to understand this

Reply:
OK - comes soon :tm: